src/cnv/tests/system/test_combine_cnmops_cnvpytor_cnv_calls.py

```diff
@@ -0,0 +1,47 @@
+import filecmp
+from os.path import join as pjoin
+from pathlib import Path
+
+import pytest
+from ugbio_cnv import combine_cnmops_cnvpytor_cnv_calls
+
+
+@pytest.fixture
+def resources_dir():
+    return Path(__file__).parent.parent / "resources"
+
+
+class TestCombineCnmopsCnvpytorCnvCalls:
+    def test_combine_cnmops_cnvpytor_cnv_calls(self, tmpdir, resources_dir):
+        input_cnmops_file = pjoin(resources_dir, "NA24385.cnmops.cnvs.chr1-2.bed")
```

Comment:
more easy and more nice is to use the pathlib.Path convention:
like:
`input_cnmops_file = resources_dir / "NA24385.cnmops.cnvs.chr1-2.bed"`
same to all pjoin below

Reply:
If I do this, tests do not pass locally. it complains it can't handle this type of object.

Reply:
but we have many such tests
like:
https://github.com/Ultimagen/ugbio-utils/blob/master/src/single_cell/tests/unit/test_single_cell_qc_pipeline.py#L90

Reply:
doesn't work locally. 
see that also in your example there is casting for str here : https://github.com/Ultimagen/ugbio-utils/blob/ced720ca16cd7db6c8c28a4d8bd9a1c4b758b6f2/src/single_cell/tests/unit/test_single_cell_qc_pipeline.py#L23

I can show you if you want.